manim/cli/render/global_options.py

```diff
@@ -36,4 +51,15 @@
         default=None,
         help="Display warnings for outdated installation.",
     ),
+    option(
+        "--enable_gui",
+        is_flag=True,
+        help="Enable GUI interaction.",
+    ),
+    option(
+        "--gui_location",
+        default="40,800",
```

Comment:
Adjusts the default here too
```suggestion
        default="0,0",
```